include/spdk/env.h

```diff
@@ -126,6 +131,8 @@ void spdk_memzone_dump(FILE *f);
 
 struct spdk_mempool;
 
+#define SPDK_MEMPOOL_CACHE_MAX_SIZE 512
```

Comment:
This value is configurable in DPDK; I don't think we can hard-code it like this in env.h.  I believe this value is only used in initialization paths that aren't performance critical, so we could add an accessor function to get this value rather than a #define.